test/integration/kubernetes_deploy_test.rb

```diff
@@ -980,7 +980,7 @@ def test_adds_namespace_labels_to_statsd_tags
 
     # We can't ensure that all the metrics we grab are from this specific test because they are running in parallel
     metrics = metrics.select { |m| m.tags.include? "namespace:#{@namespace}" }
-    assert_equal 4, metrics.count
+    assert_equal 5, metrics.count
```

Comment:
Bumped `metrics.count` assertion to 5 since we are now emitting an additional event metric to `statsd`.

Reply:
Is there an easy way to identify the event? If so, let's add an assertion about the event as well.